.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_14" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
it was JDK_11 originally, so revert it to this.
first you changes JDK_11 to JDK_14 and now to JDK_1_8.

see this https://github.com/Ni3verma/Safe-Box/pull/18#discussion_r657905700

Reply:
if it's automatically changing then I have to way, lets connect on slack.
TBH i am not sure what these params mean in misc file and build.gradle file. so I cant tell you to change in gradle file.